src/bin/nydusd/main.rs

```diff
@@ -343,6 +343,26 @@ fn append_services_subcmd_options(app: App<'static, 'static>) -> App<'static, 's
                 .help("Tag to identify the fscache daemon instance")
                 .takes_value(true)
                 .requires("fscache"),
+        )
+        .arg(
+            Arg::with_name("fscache-threads")
+                .long("fscache-threads")
+                .default_value("1")
```

Comment:
Can we provide a suggested default value here, for example processor number?

Reply:
I think it's a little to over-provision for one worker thread per processor.
I feel 1/2/4/8 should be enough.